.travis.yml

```diff
@@ -59,7 +59,7 @@ script:
   # Test ruby gem
   - gem install gem-path --no-ri --no-rdoc
   - cd $HOME/gems/symengine-0.1.0/
-  - bundle exec rspec
+  - bundle exec rspec --require ./spec/spec_helper.rb
```

Comment:
As we are testing against installed gem, and gem files are not including `.rspec`, we need to specify this directly.